source/documentation/managing_apps/restarts.md

```diff
@@ -0,0 +1,13 @@
+## App restarts
+
+Cloud Foundry will restart your app due to circumstances outside of your direct control, such as:
+
+- platform upgrades
+- failure of the underlying hardware platform
+- applying operating system patches to virtual machines that host app instances
+
+Cloud Foundry sends a termination signal (SIGTERM) to any apps it wants to restart. To shut down cleanly, apps must finish any requests within 10 seconds of receiving the termination signal. If the app does not respond in time, Cloud Foundry will force-terminate the app through a SIGKILL signal without shutting the app down cleanly, and requests may fail. 
```

Comment:
Can be more explicit what the tenant should do if they have processes that run for more than 10 seconds.